### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by BridgeAR**
Update all dependencies including the devDependencies besides bluebird.
Btw: Why does sequelize need coffee-script as a dependency?

### CODE REVIEW

**Comment by mickhansen** [comment]
coffee-script must be leftover from the cli, doubt we need it anymore.

File: package.json [hunk 1]
```diff
@@ -47,16 +47,16 @@
     "chai-as-promised": "^4.3.0",
     "chai-datetime": "~1.3.0",
     "chai-spies": "~0.5.1",
-    "coffee-script": "~1.7.1",
+    "coffee-script": "~1.9.1",
     "commander": "^2.6.0",
     "continuation-local-storage": "^3.1.4",
-    "dox": "0.5.0",
+    "dox": "0.7.1",
```
Comment (mickhansen) [inline package.json]: Have you tested that doc generation still works?

Comment (janmeier) [inline package.json]: Testing dox right now - seems to work :)

**Comment by BridgeAR** [comment]
I'd remove coffee-script but there's still the reference in the deprecated? migration part:
```
/home/ruben/fintura/sequelize/lib/migration.js:
   56          try {
   57            // 1.7.x compiler registration
   58:           require('coffee-script/register');
   59          } catch (e) {
   60            try {
   61              // Prior to 1.7.x compiler registration
   62:             require('coffee-script');
   63            } catch (e) {
   64:             console.log('You have to add \"coffee-script\" to your package.json.');
   65              process.exit(1);
   66            }
3 matches in 1 file
```
When do you want to remove the migrator?

**Comment by mickhansen** [comment]
Not sure, we need to add a deprecation notice for it first. So probably 2.2

File: package.json (hunk 1 above)
Comment (BridgeAR) [inline package.json]: I had a look at the changelogs and couldn't find any bc's.

**Comment by BridgeAR** [comment]
Let's do that in another PR in that case. I guess this is ready to merge.

**Pull request merged**